libs/ui/src/lib/terms-of-use/components/terms-of-use-button/index.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+
+import { localization } from "@catalog-frontend/utils";
+import { Button } from "@digdir/designsystemet-react";
+import { useRouter } from "next/navigation";
+
+type TermsOfUseButtonProps = {
+    url: string;
+  };
+
+export const TermsOfUseButton = ({ url }: TermsOfUseButtonProps) => {
+    const router = useRouter();
+    
+    const handleGotoTermsOfUse = () => {
+        router.push(url);
+      };
+
+    return (
+        <Button
+          size='sm'
+          onClick={handleGotoTermsOfUse}
+        >
+          {localization.termsOfUse.gotoTermsOfUse}
+        </Button>
```

Comment:
Tror man burde bruke `<Link>` wrappet i button her for å få riktig WCAG. Tror det finnes en komponent som heter "LinkButton" eller noe sånt (evt se designsystemet under "Knapp som lenke")

Reply:
894a52547aed20e9483a0bec052541ab3644a32a